client/lib/domains/dns/index.js

```diff
@@ -126,7 +130,57 @@ function getFieldWithDot( field ) {
 	return ( typeof field === 'string' && field.match( /^([a-z0-9-_]+\.)+\.?[a-z]+$/i ) ) ? field + '.' : field;
 }
 
-module.exports = {
-	validateAllFields,
-	getNormalizedData
+function isWpcomRecord( record ) {
+	return startsWith( 'wpcom:', record.id );
+}
+
+function filterRootARecords( domain, records ) {
```

Comment:
A general request for this file - please use `domain` when the variable holds an object representing the domain and `domainName` when it's just a string representation of the domain. @umurkontaci recently introduced this consistency in #3582 and it'd be awesome if we can stick to it. It's a recent change (and there's no P2 about it) so you might have missed it.
@umurkontaci, now that I think about it, README.md in `lib/domains` should have been updated as part of that PR to document the consistent nomenclature :disappointed: